Shared (App)/Services/TransactionService.swift

```diff
@@ -83,19 +83,19 @@ final class TransactionService: TransactionFetchable {
     
     @MainActor
     func fetchTransactions(network: Network, address: Address) async throws -> [TransactionGroup] {
-        async let alchemyOutgoingTransactions = self.alchemyClient!.alchemyAssetTransfers(fromBlock: .genesis,
-                                                                                          toBlock: .latest,
-                                                                                          fromAddress: address)
-        async let alchemyIncomingTransactions = self.alchemyClient!.alchemyAssetTransfers(fromBlock: .genesis,
-                                                                                          toBlock: .latest,
-                                                                                          toAddress: address)
-        async let covalentTransactions = self.covalentClient!.getTransactions(network: .ethereum, address: address)
+//        async let alchemyOutgoingTransactions = self.alchemyClient!.alchemyAssetTransfers(fromBlock: .genesis,
+//                                                                                          toBlock: .latest,
+//                                                                                          fromAddress: address)
+//        async let alchemyIncomingTransactions = self.alchemyClient!.alchemyAssetTransfers(fromBlock: .genesis,
+//                                                                                          toBlock: .latest,
+//                                                                                          toAddress: address)
+//        async let covalentTransactions = self.covalentClient!.getTransactions(network: .ethereum, address: address)
         async let unmarshalTransactions = self.unmarshalClient!.getTransactions(address: address)
```

Comment:
Is this force unwrap safe?

Reply:
I think this method was just a quick implementation for demo purposes. Once we agree on how to handle txs we can rewrite to handle it properly.